app/javascript/src/utilities/utils.js

```diff
@@ -2,20 +2,21 @@
 import React from 'react'
 
 function CSRFToken ({win = window}: {win?: Window}) {
-  const hasCSRFParam = win.document.head.querySelector(`meta[name='csrf-param']`)
-  if (!hasCSRFParam) {
-    console.log('CSRF issue: No csrf-param present in meta tags!')
-    return null
-  }
   const getMetaContent = meta => win.document.head.querySelector(`meta[name~=${meta}][content]`).content
-  const props = {
-    name: getMetaContent('csrf-param'),
-    value: getMetaContent('csrf-token'),
-    type: 'hidden'
+
+  try {
+    const props = {
+      name: getMetaContent('csrf-param'),
+      value: getMetaContent('csrf-token'),
+      type: 'hidden'
+    }
+    return (
+      <input {...props} />
+    )
+  } catch (error) {
+    console.error(error)
+    return null
```

Comment:
@didierofrivia I think using try... catch we don't need anymore the hasCSRFParam var.
And the error is a lot more explicit :)